Cargo.toml

```diff
@@ -156,7 +156,11 @@ hyper = { version = "0.14", features = ["server"] }
 ip_network = "0.4"
 lazy_static = { version = "1.4", default-features = false }
 once_cell = "1.19"
-log = { version = "0.4", features = ["std", "kv_std"] }
+log = { version = "0.4", features = [
+  "std",
+  "kv_std",
+  "release_max_level_debug",
```

Comment:
bsaically, trace! expands to an `if level >= log::STATIC_MAX_LEVEL && get_level(target)`, and STATIC_MAX_LEVEL is a const which expands to debug when this feature flag is set + when `debug_assertions` is off

so, in production & release builds, trace! calls will be optimized out as deadcode, their arguments won't even be evaludated.

Reply:
ah, your question is about info

i think we should keep debug logs in prod builds just in case, and i don't see the perf benefits of leaving them in